keyboards/planck/rev6/config.h

```diff
@@ -139,4 +139,8 @@
 #define WS2812_DMA_STREAM STM32_DMA1_STREAM2
 #define WS2812_DMA_CHANNEL 2
 
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED true
+#endif
```

Comment:
This will allow users to override from keymap, should they feel so inclined.

```suggestion
#ifndef RGB_DISABLE_WHEN_USB_SUSPENDED
#    define RGB_DISABLE_WHEN_USB_SUSPENDED true
#endif
```